packages/react-components/review/beta/react-components.api.md

```diff
@@ -1253,6 +1253,9 @@ export const _LocalVideoTile: React_2.MemoExoticComponent<(props: {
     personaMinSize?: number | undefined;
 }) => JSX.Element>;
 
+// @beta
+export type LocalVideoTileMode = 'mobile' | 'desktop' | 'hidden' | 'responsive';
```

Comment:
`desktop`/`mobile`/`responsive` are vague -- what does it mean to have a tile mobile vs desktop? Could we make the naming more inline with the behavior instead of the platform we expect them to be on?

Reply:
I like the updates around using the aspect ratio, but for 'matchStream' I think is a little misleading since it isn't changing to match the stream size, its still responding to the size observers making it responsive today. would `'16:9' | '9:16' | 'responsive'` be appropriate?

Reply:
I guess my question is why is it responsive? Wouldn't it make more sense to always match the incoming stream? (If they are on mobile and portrait vs landscape, their stream will then be indicative of the mobile size)

Reply:
On smaller screens the tile itself could be smaller but that would be tangential to the aspect ratio

Reply:
Oh because we can't always match it if there video is off, hmm

Reply:
actually match stream makes more sense thinking about it will make these updates.